nbodykit/algorithms/convpower.py

```diff
@@ -669,21 +669,23 @@ def normalization(self, name, alpha):
 
         if name+'.norm' not in self.attrs:
 
-            # first/second meshes for "name" (data or randoms)
-            first = self.first[name]
-            second = self.second[name]
+            # the selection (same for first/second)
+            sel = self.first[name][self.first.selection]
```

Comment:
is self.first.selection an array like?

Reply:
``sel`` is a dask array. ``self.first.selection`` is the name of the selection column, and ``self.first[name]`` is a CatalogMesh holding the data or randoms species